spring-jdbc/src/main/java/org/springframework/jdbc/core/metadata/GenericTableMetaDataProvider.java

```diff
@@ -76,6 +76,8 @@
 	/** Collection of TableParameterMetaData objects. */
 	private List<TableParameterMetaData> tableParameterMetaData = new ArrayList<>();
 
+	/** the string used to quote SQL identifiers. */
+	private String identifierQuoteString = "";
```

Comment:
There is at least one RDBMS (MS SQL) which uses different characters for opening and closing. ( [identifier] )

Reply:
Could you please provide link to documentation where it is mentioned ? I could only find this https://docs.microsoft.com/en-us/sql/odbc/reference/develop-app/quoted-identifiers?view=sql-server-ver15. 
We are getting this identifier to quote SQL from database metadata https://docs.oracle.com/javase/6/docs/api/java/sql/DatabaseMetaData.html#getIdentifierQuoteString%28%29. If this class does not support case where there are 2 different characters to quote SQL identifiers, we also could not.

Reply:
Thanks for the links to the documentation. It seems that the driver takes care about switching on the support of quoted identifiers (QUOTED_IDENTIFIER) so the change will work with MS SQL as well.